util/jsonutil/keywordsutil_test.go

```diff
@@ -0,0 +1,34 @@
+package jsonutil
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestKeywordsUtilUnmarshalJSON(t *testing.T) {
+	type Keywords struct {
+		Keywords KeywordsString `json:"keywords"`
+	}
+
+	t.Run("dynamic json", func(t *testing.T) {
```

Comment:
Spaces are not allowed in test names. The test framework will replace them. Consider naming this something like `dynamic-json` so the test framework doesn't need to change the name.

Reply:
addressed in [580c047](https://github.com/prebid/prebid-server/pull/2649/commits/580c0474c9e2861efb838335957f5556bc87b16c)